radix-engine/src/wasm/constants.rs

```diff
@@ -1,7 +1,7 @@
 pub const RADIX_ENGINE_FUNCTION_INDEX: usize = 0;
 pub const RADIX_ENGINE_FUNCTION_NAME: &str = "radix_engine";
-pub const USE_TBD_FUNCTION_INDEX: usize = 1;
-pub const USE_TBD_FUNCTION_NAME: &str = "gas";
+pub const CONSUME_COST_UNIT_FUNCTION_INDEX: usize = 1;
+pub const CONSUME_COST_UNIT_FUNCTION_NAME: &str = "gas";
```

Comment:
"cost unit"

Reply:
No way to change this without forking the library.